pkg/processor/build/builder.go

```diff
@@ -657,6 +663,42 @@ func (b *Builder) createProcessorDockerfile() (string, error) {
 	return processorDockerfilePathInStaging, nil
 }
 
+// replace known keywords in docker command list with directives
+// runTime can be nil - used for injection testing
+func (b *Builder) preprocessBuildCommands(commands []string, runTime string) ([]string, error) {
+	var processedCommands []string
+
+	if runTime == "" {
+		runTime = time.Now().String()
+	}
+	knownKeywords := map[string]string{
+		"noCache": fmt.Sprintf("RUN echo %s > /dev/null", runTime),
+	}
+
+	commentPattern, err := b.getRuntimeCommentPattern(b.options.FunctionConfig.Spec.Runtime)
```

Comment:
You're looking for a comment pattern but don't need to. Users just write `@nuclio.noCache`, they don't prefix this with the comment pattern of the function's runtime